python/graphscope/interactive/query.py

```diff
@@ -27,6 +27,8 @@
 from graphscope.framework.dag import DAGNode
 from graphscope.framework.dag_utils import gremlin_to_subgraph
 
+from neo4j import GraphDatabase
```

Comment:
Add try except, user may not want to install neo4j if they just need gremlin.
try:
  from neo4j ...
except:
  pass
  # cypher query is disabled